docker-entrypoint.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
```

Comment:
great work, may I ask for a super-small addition? .. could you pls. add headers to the files created (maybe also the docker file) with permission notice and disclaimer:

# Copying and distribution of this file, with or without modification,
# are permitted in any medium without royalty provided the copyright
# notice and this notice are preserved.  This file is offered as-is,
# without any warranty.

like here (scroll down):
https://github.com/fossology/fossology/wiki/Coding-Style